stdlib/gc.mli

```diff
@@ -145,17 +145,46 @@ type control =
     mutable allocation_policy : int;
     [@ocaml.deprecated_mutable
          "Use {(Gc.get()) with Gc.allocation_policy = ...}"]
-    (** The policy used for allocating in the heap.  Possible
-        values are 0 to 2.  0 is the original next-fit policy,
-        usually fast, but can result in fragmentation.  1 is the
-        first-fit policy, which can be slower in some cases but
-        can be better for programs with fragmentation problems.
-        2 is the best-fit allocation policy, the best for avoiding
-        fragmentation.
-        Note that changing the allocation policy at run-time forces
+    (** The policy used for allocating in the major heap.
+        Possible values are 0, 1 and 2.
+
+        - 0 is the next-fit policy, which is usually fast but can
+          result in fragmentation.
+
+        - 1 is the first-fit policy, which avoids fragmentation but
+          is sensibly slower in some real-world use-cases (it is only
+          useful on programs exhibiting fragmentation issues with
+          next-fit)
+
+        - 2 is the best-fit policy, which is empirically better than
+          next-fit and first-fit: in our experiments it is faster and
+          uses less memory than both.
```

Comment:
`emprically better` sounds redundant with `in our experiments`. Also, the example below doesn't support the assertion
>it is faster and uses less memory than both.

 since best-fit is not strictly faster than next-fit and don't use strictly less memory than `first-fit`. Maybe, we could rephrase it to something like

> it combines the speed of next-fit with the fragmentation of first-fit

Reply:
Damien's number do show that best-fit tends to be both faster and less-memory-hungry than either modes, although that doesn't really show here with just this one run. (Note in particular that given the tradeoff between reclamation frequency and throughput, it doesn't really make sense to say that one strategy is "faster than first-fit but uses just as much memory" , as for most workflows trade speed for space and get something that is slightly slower than before (but still faster) and does use less memory.) Let me just replace the numbers I gave with a different run where the result is closer to the impression I want to give :-)

Reply:
Updating the example to be coherent with the claims is fine with me.